modules/ppcp-blocks/resources/js/checkout-block.js

```diff
@@ -103,8 +103,9 @@ if ( blockEnabled ) {
 			edit: descriptionElement,
 			placeOrderButtonLabel: config.placeOrderButtonText,
 			ariaLabel: config.title,
-			canMakePayment: () => {
-				return true;
+			canMakePayment: ( cartData ) => {
+				const totals = cartData?.cartTotals?.total_price;
+				return parseInt( totals ) > 0;
```

Comment:
I would rename to something like `total`, `totals` sounds like an array.

```suggestion
				const total = cartData?.cartTotals?.total_price;
				return parseInt( total ) > 0;
```

Reply:
Thanks for the suggestion. I agree. I adjusted the name of the variable.